README.md

```diff
@@ -143,21 +142,19 @@ some steps for setting that up.
 3. In the deploy directory, log into Pulumi CLI and create a new stack.
 4. Run this in the deploy directory: `pulumi config set cobra:cobra_domain your_domain.com`. 
    Ensure you own the domain you want to use.
-5. Do the same for cobra:nisei_domain. If you don't like the defaults for cobra:region and cobra:size, you can set them
-   to slug values shown here: https://slugs.do-api.dev/.
-6. If you have NetrunnerDB client credentials, encrypt them with these commands:
+5. If you have NetrunnerDB client credentials, encrypt them with these commands:
```

Comment:
The information about the slug values is still relevant without the extra domain. I'd still find the link useful.

Reply:
Done.